lib/db.js

```diff
@@ -724,6 +750,16 @@ module.exports = function (
     return this.pool.post('/account/' + account.uid.toString('hex') + '/verifyEmail')
   }
 
+  DB.prototype.verifyToken = function (tokenVerificationId, accountData) {
+    log.trace({ op: 'DB.verifyToken', tokenVerificationId: tokenVerificationId })
+    return this.pool.post('/token/' + tokenVerificationId.toString('hex') + '/verify',
+        unbuffer(
+          accountData,
+          'inplace'
+        )
+    )
+  }
+
```

Comment:
In fact, I suggest updating the npm-shrinkwrap file as part of this PR, so that it pulls in an appropriate commit of fxa-auth-db-mysql with the latest changes.